Resolve cli flag (#1616)
facebookresearch/hydra: website/docs/tutorials/basic/running_your_app/5_debugging.md

...
```

> If the configuration contains any interpolations, these will not be resolved by default.
> To resolve variable interpolations before the config is printed,
> use the `--resolve` flag in addition to the `--cfg` flag.
> 

Reviewer: ```suggestion
By default, config interpolations are not resolved. To print resolved config use the `--resolve` flag in addition to the `--cfg` flag
```
